django_cloud_tasks/tasks.py

```diff
@@ -261,3 +262,42 @@ def __client(self):
         )
 
         return CloudPublisher(publisher_options=publisher_options)
+
+
+class RoutineTask(Task):
+    def run(self, routine_id: int):
```

Comment:
This whole execution should be inside a lock since there's too many possible triggers.